### PR TITLE
CSP-214 Implements standard review page

### PR DESCRIPTION
**Pull request by DevenKShah**

### CODE REVIEW

File: src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/AddAStandard/ReviewAndSaveStandardController.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            if (sessionModel == null) return redirectResult;
+            sessionModel.CancelLink = GetUrlWithUkprn(RouteNames.ViewStandards);
+            return View(ViewPath, sessionModel);
+        }
+
+        [HttpPost]
+        [Route("{ukprn}/standards/add/save-standard", Name = RouteNames.PostAddStandardReviewStandard)]
+        public IActionResult SaveStandard()
+        {
+            return RedirectToRouteWithUkprn(RouteNames.ViewStandards);
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/AddAStandard/ReviewAndSaveStandardController.cs:35]: There's on coverage on this, but there's also no logic, so coverage not necessary imo. I assume a future story will address this.

File: src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderCourseLocations/ProviderCourseLocationViewModel.cs [hunk 2]
```diff
@@ -44,17 +26,14 @@ public string DeliveryOption()
...
-                HasDayReleaseDeliveryOption = providerCourseLocation.HasDayReleaseDeliveryOption,
-                HasBlockReleaseDeliveryOption = providerCourseLocation.HasBlockReleaseDeliveryOption,
-                RegionName = providerCourseLocation.RegionName
+                DeliveryMethod = new DeliveryMethodModel 
+                { 
+                    HasDayReleaseDeliveryOption = providerCourseLocation.HasDayReleaseDeliveryOption,
+                    HasBlockReleaseDeliveryOption = providerCourseLocation.HasBlockReleaseDeliveryOption
+                },
+                RegionName = providerCourseLocation.RegionName,
+                SubregionName = providerCourseLocation.SubregionName
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderCourseLocations/ProviderCourseLocationViewModel.cs:35]: I can't find any coverage for RegionName and SubregionName, can you add it if appropriate?

File: src/SFA.DAS.Roatp.CourseManagement.Web/Models/DeliveryMethodModel.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            if (HasDayReleaseDeliveryOption.GetValueOrDefault())
+            {
+                return DayRelease;
+            }
+            if (HasBlockReleaseDeliveryOption.GetValueOrDefault())
+            {
+                return BlockRelease;
+            }
+            return string.Empty;
+        }
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Models/DeliveryMethodModel.cs:25]: I would suggest add coverage for ToSummary

**Review by MarkFCain — DISMISSED** [review]
Approved, logic is good, some requests for coverage additions

File: src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/AddAStandard/ReviewAndSaveStandardController.cs (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/AddAStandard/ReviewAndSaveStandardController.cs:35]: Your assumption is correct.

File: src/SFA.DAS.Roatp.CourseManagement.Web/Models/DeliveryMethodModel.cs (hunk 3 above)
Comment (DevenKShah) [reply src/SFA.DAS.Roatp.CourseManagement.Web/Models/DeliveryMethodModel.cs:25]: Tests for ToSummary are there in `ProviderCourseLocationViewModelTests`

Comment (DevenKShah) [reply src/SFA.DAS.Roatp.CourseManagement.Web/Models/DeliveryMethodModel.cs:25]: Done

File: src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderCourseLocations/ProviderCourseLocationViewModel.cs (hunk 2 above)
Comment (DevenKShah) [reply src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderCourseLocations/ProviderCourseLocationViewModel.cs:35]: Added.

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**